src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -667,10 +674,65 @@ trait BeamHelper extends LazyLogging {
     * @param outputDirectory output folder where full configs will be generated
     */
   private def writeFullConfigs(config: TypesafeConfig, outputDirectory: String): Unit = {
-    val configConciseWithoutJson = config.root().render(ConfigRenderOptions.concise().setFormatted(true).setJson(false))
+
+    val configResolveOptions = ConfigResolveOptions.defaults().setAllowUnresolved(true)
+
+    val templateConf = ConfigFactory
+      .parseResources("beam-template.conf")
+      .withoutPath("matsim.modules.vehicles.vehiclesFile")
```

Comment:
Shall it be extracted to something like `excludeList`, so new settings can be easily added to that list?

Reply:
added a list which consist all excluded config param